docs/conf.py

```diff
@@ -512,8 +512,8 @@ def get_colors(color_t: str):
     ("cpp:identifier", "my_ns3"),
     ("cpp:identifier", "cpp_apigen_demo"),
     # Example JavaScript types
-    ("js:func", "string"),
-    ("js:func", "SomeError"),
+    ("js:class", "string"),
+    ("js:class", "SomeError"),
```

Comment:
reverting this. Demo docs declare these entities as JS function sugnatures
```suggestion
    ("js:func", "string"),
    ("js:func", "SomeError"),
```